outboundSender.go

```diff
@@ -103,6 +107,8 @@ type OutboundSenderFactory struct {
 
 	// DisablePartnerIDs dictates whether or not to enforce the partner ID check.
 	DisablePartnerIDs bool
+
+	QueryLatency metrics.Histogram
```

Comment:
Do we need this field?

Reply:
I think it's needed. This field is used in senderWrapper.go to add the "url" label to the histogram